Flow.Launcher/Themes/Metro Server.xaml

```diff
@@ -1,39 +1,39 @@
 <ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
-	<ResourceDictionary.MergedDictionaries>
+    <ResourceDictionary.MergedDictionaries>
         <ResourceDictionary Source="pack://application:,,,/Themes/Base.xaml"></ResourceDictionary>
-	</ResourceDictionary.MergedDictionaries>
-	<Style x:Key="QueryBoxStyle" BasedOn="{StaticResource BaseQueryBoxStyle}" TargetType="{x:Type TextBox}">
-		<Setter Property="Background" Value="#ffffff"/>
-		<Setter Property="Foreground" Value="#000000" />
-	</Style>
+    </ResourceDictionary.MergedDictionaries>
+    <Style x:Key="QueryBoxStyle" BasedOn="{StaticResource BaseQueryBoxStyle}" TargetType="{x:Type TextBox}">
```

Comment:
this theme(non edited), it works but have design problem. (inputbox full height)